.github/CODEOWNERS

```diff
@@ -30,175 +30,257 @@
+# ServiceLabel: %Event Grid
+/sdk/eventgrid/ @sarangan12
 
 # PRLabel: %Event Hubs
-/sdk/eventhub/ @jeremymeng @deyaaeldeen @HarshaNalluru
+# ServiceLabel: %Event Hubs
+# AzureSdkOwners: @deyaaeldeen @jeremymeng 
+/sdk/eventhub/ @deyaaeldeen @jeremymeng @HarshaNalluru
 
 # PRLabel: %Azure.Identity
-/sdk/identity/ @KarishmaGhiya @maorleger @witemple-msft @schaabs @Azure/azure-sdk-write-identity
+# ServiceLabel: %Azure.Identity
+# AzureSdkOwners:  @KarishmaGhiya @maorleger
+/sdk/identity/ @KarishmaGhiya @maorleger @schaabs @Azure/azure-sdk-write-identity
 
 # PRLabel: %Iot
+# ServiceLabel: %Iot
+# AzureSdkOwners:  @xirzec
 /sdk/iot/ @xirzec
 
 # PRLabel: %KeyVault
+# ServiceLabel: %KeyVault
+# AzureSdkOwners:  @timovv @maorleger
 /sdk/keyvault/ @timovv @maorleger @Azure/azsdk-keyvault
 
 # PRLabel: %OpenTelemetryInstrumentation
-/sdk/instrumentation/ @maorleger @joheredi @mpodwysocki
+# ServiceLabel: %OpenTelemetryInstrumentation
+# AzureSdkOwners: @maorleger @mpodwysocki
+/sdk/instrumentation/ @maorleger @mpodwysocki
 
 # PRLabel: %Service Bus
+# ServiceLabel: %Service Bus
+# AzureSdkOwners: @jeremymeng @deyaaeldeen @HarshaNalluru
 /sdk/servicebus/ @jeremymeng @deyaaeldeen @HarshaNalluru
 
 # PRLabel: %Storage
-/sdk/storage/ @EmmaZhu @XiaoningLiu @jeremymeng @vinjiang @jiacfan @ljian3377
+# ServiceLabel: %Storage
+/sdk/storage/ @EmmaZhu @XiaoningLiu @vinjiang @jiacfan @ljian3377
 
 # PRLabel: %Synapse
+# ServiceLabel: %Synapse
+# AzureSdkOwners: @joheredi
 /sdk/synapse/ @joheredi
 
 # PRLabel: %Tables
+# ServiceLabel: %Tables
+# AzureSdkOwners: @joheredi
 /sdk/tables/ @joheredi
 
 # PRLabel: %Purview
+# ServiceLabel: %Purview
+# AzureSdkOwners: @qiaozha
 /sdk/purview/ @qiaozha
 
 # PRLabel: %Farmbeats
+# ServiceLabel: %Farmbeats
+# AzureSdkOwners: @joheredi
 /sdk/agrifood/ @joheredi
 
 # PRLabel: %ConfidentialLedger
+# ServiceLabel: %ConfidentialLedger
+# AzureSdkOwners: @joheredi
 /sdk/confidentialledger/ @joheredi
 
 # PRLabel: %DocumentTranslator
+# ServiceLabel: %DocumentTranslator
+# AzureSdkOwners: @joheredi
 /sdk/documenttranslator/ @joheredi
 
 # PRLabel: %WebPubSub
-/sdk/web-pubsub/ @vicancy @xirzec @bterlson
+# ServiceLabel: %WebPubSub
+/sdk/web-pubsub/ @vicancy
 
 # PRLabel: %EngSys
+# ServiceLabel: %EngSys
+# AzureSdkOwners: @ckairen @mikeharder @weshaggard @benbp
 /sdk/template/ @ckairen @mikeharder @weshaggard @benbp
 
+# PRLabel: %test-utils
+# ServiceLabel: %test-utils
+# AzureSdkOwners: @HarshaNalluru @timovv
 /sdk/test-utils/ @HarshaNalluru @timovv
 
 # PRLabel: %Cognitive - Text Analytics
-/sdk/textanalytics/ @quentinRobinson @wangyuantao @minhanh-phan @deyaaeldeen @witemple-msft
+# ServiceLabel: %Cognitive - Text Analytics
+/sdk/textanalytics/ @quentinRobinson
 
 # PRLabel: %Cognitive - Language
-/sdk/cognitivelanguage/ @quentinRobinson @wangyuantao @minhanh-phan @deyaaeldeen
+# ServiceLabel: %Cognitive - Language
+/sdk/cognitivelanguage/ @quentinRobinson
+
+# ServiceLabel: %OpenAI
+# ServiceOwners: @glharper
 
```

Comment:
Originally I tried to combine this but the linter complained:

```
Source block error.
Source Block Start: 232
  # PRLabel: %OpenAI
  # ServiceLabel: %OpenAI
  # AzureSdkOwners: @deyaaeldeen @minhanh-phan
  # ServiceOwners: @glharper
  /sdk/openai/ @glharper @deyaaeldeen @minhanh-phan
Source Block End: 236
  -ServiceLabel cannot be part of a block with, /<NotInRepo>/ or ServiceOwners, and a source path/owner line.

```

I *think* we could do what you are saying, but if we add `AzureSdkOwners` there then you would always be on the hook for triage and SLA would count against Azure SDK rather than it being owned by Glenn.

Reply:
Sounds good, we can always go back and change it if there is a need to.

Reply:
I wonder if the magic cheat is to have it be %OpenAI %Service Attention for those cases?